pom.xml

```diff
@@ -68,7 +75,6 @@
             <id>springboot</id>
             <activation>
                 <file>
-                    <!-- Just a way to identify a Spring Boot application -->
```

Comment:
I think we could keep this comment. Don't you?

Reply:
Fixed in c4c688f0202237231ff7b07e7ccb71487bc77458